issue_template.md

```diff
@@ -28,7 +28,7 @@ Replace this line with URL(s) for issue
 
 <!-- Which adblock lists are you're using? -->
 
-### Basic OS and browser informaations
+### Basic OS and browser information's
```

Comment:
Does this need an _'s_?

Reply:
As I recall from my time in school yes, as there are more than one information

But hey, that is what the PR is for, finding errors before pushing.

PS: For your emoji... I'm sorry there isn't a :sob: `:sob:` but still training in my new keyboard and the other new one just like generating doubles. Pretty annoying actually